Premier league.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<ol><li>Rename the dict <code>all_teams</code>  - make it more verbose.</li><li>Row 7 could be replaced with a simple <code>mean()</code>  operation</li><li>So we see Barnsley has a low value. Please add the distance of the Barnsley from the mean in sigmas:</li></ol><p><code>(values.mean() - values.min()) / values.std()</code> </p><p>is it large ? </p><p>This outlier could be caused by a team that played only a few games - is it the case here ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/A4Vision/games-course/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
<p>No, I calculated the chance for draw game in "normalize" (In relation to the number of games of each team).</p>

Reply:
<p>Indeed, you "normalized" the calculation by dividing.</p><p>However, I meant something else.</p><p> </p><p>I meant that the estimation of this probability could be too rough in order to compare the numbers.</p><p>Assume for example, that we had a team that played exactly one game. That team would have an estimated probability to draw of either 1 or 0. This number says nothing about its real probability to draw and it cannot be used for statistical inference.</p><p>Now, I repeat my question </p><p> </p><p>This outlier could be caused by a team that played only a few games - is it the case here ?</p><p> </p>